spec/jobs/notification/reopen_snoozed_notifications_job_spec.rb

```diff
@@ -13,11 +13,15 @@
   context 'when called' do
     it 'reopens snoozed notifications whose snooze until has passed' do
       described_class.perform_now
+      time_after_job = Time.current
```

Comment:
rather than doing this.. assign the value in `snoozed_until` field of `snoozed_till_5_minutes_ago` notification into a variable